server/handlers.go

```diff
@@ -1029,13 +1029,13 @@ func requestTokensHandler(client eth.LivepeerEthClient) http.Handler {
 		}
 
 		backend := client.Backend()
-		blk, err := backend.BlockByNumber(r.Context(), nil)
+		h, err := backend.HeaderByNumber(r.Context(), nil)
```

Comment:
Probably better to only fetch the block header as opposed to the entire block here anyway because we don't care about most of the data in the block anyway and we can just get the information we need from the slimmer header.